### ISSUE TITLE
jx create quickstarts fails when using the same project type twice

### ISSUE THREAD

**Issue by tomfrenken** [jenkins-x/jx#6495]
### Summary 
When using `jx create quickstart` on for example `spring-boot-rest-prometheus`, then deleting the application with `jx delete application` and then running `jx create quickstart` with a `spring-boot-rest-prometheus` project again, you receive the following error message (most likely also if you don't delete the application)

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-deploy-plugin:2.8.2:deploy (default-deploy) on project spring-boot-rest-prometheus: Failed to deploy artifacts: Could not transfer artifact com.github.jenkinsx.quickstarts:spring-boot-rest-prometheus:jar:0.0.1 from/to local-nexus (http://nexus/repository/maven-releases/): Transfer failed for http://nexus/repository/maven-releases/com/github/jenkinsx/quickstarts/spring-boot-rest-prometheus/0.0.1/spring-boot-rest-prometheus-0.0.1.jar 400 Repository does not allow updating assets: maven-releases -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException

Pipeline failed on stage 'from-build-pack' : container 'step-build-mvn-deploy'. The execution of the pipeline has stopped.
```
### Steps to reproduce the behavior
Run `jx create quickstart` with the same project (e.g. `pring-boot-rest-prometheus`) twice.

### Expected behavior
No errors

### Actual behavior
Errors

### Jx version

The output of `jx version` is:

```
jx                 2.0.1117
Kubernetes cluster v1.13.11-gke.14
kubectl            v1.17.0
helm client        Client: v2.16.1+gbbdfe5e
git                2.24.1
Operating System   Mac OS X 10.15.2 build 19C57
```

### Jenkins type

- [X] Serverless Jenkins X Pipelines (Tekton + Prow)
- [ ] Classic Jenkins

**Comment by hferentschik** [comment]
Can you provide some more context and maybe an example to reproduce this?

**Comment by tomfrenken** [comment]
Sure! So I've came along this issue when I was creating a `spring-boot-rest-prometheus` for the 2nd time and suddenly got the errors mentioned above, to reproduce this I recreated my cluster, reinstalled jx and created the `spring-boot-rest-prometheus` application twice in a row, the first time worked fine, the 2nd threw the above error.
To add to this, when I'm now building the `spring-boot-rest-prometheus-java11` it is also throwing errors - however, the `spring-boot-http-gradle` was still building fine.
